### DISCUSSION TITLE
Replication privilegs not present when importing from MariaDB

### DISCUSSION THREAD

**Discussion by aokigit** [planetscale/discussion#303] [Q&A] [answered]
I'm trying to import my database on PlanetScale that's currently sitting on a MariaDB server, but I'm stuck at this step getting the following error:

> external database does not have the required user grants: required privileges [REPLICATION CLIENT] are not present on the database user

I've googled for hours on how to fix this, but I haven't found any working solution so far. The database user I'm using during the import process has the correct privileges and I've also tried creating a new account without luck.

I also found out, that the REPLICATION CLIENT privilege was renamed to BINLOG MONITOR with [MariaDB 10.5](https://mariadb.com/kb/en/changes-improvements-in-mariadb-105/). 
I've tried to grant both the REPLICATION CLIENT and BINLOG MONITOR privileges to the user, but it doesn't seem to make a difference.

Any help is appreciated.

**Comment by jgreet** [comment, marked as answer]
Hi @aokigit, did you find this thread already? https://github.com/planetscale/discussion/discussions/254

The situation is basically unchanged from there, MariaDB is only partially supported but we can disable the Connection Test phase of the import if you'd like to make an attempt anyway. Just [open a ticket](https://support.planetscale.com/hc/en-us/requests/new) and tell us your organization name.

**Comment by aokigit** [comment]
Hi, 
I unfortunately found it an hour after I've made this topic, my apologies. I've sorted out the issue by migrating the database from MariaDB to MySQL and everything went well so far!

**Reply by jgreet** [reply]
Glad you found another way! I think we'll have the privilege checks updated for MariaDB soon but it still won't be as reliable as importing from MySQL.
